includes/admin/customers/class-customer-addresses-table.php

```diff
@@ -216,7 +216,7 @@ public function column_customer( $item ) {
 	 */
 	public function column_cb( $item ) {
 		$customer = edd_get_customer_by( 'id', $item['customer_id'] );
-		$name     = isset( $customer->name ) ? $customer->name : $customer->email;
+		$name     = isset( $customer->name ) ? $customer->name : 'Address ID: ' . $item['id'];
```

Comment:
Here's sort of what I was thinking:

Before this entire line, set `$name` to be the address ID (similar to what you've done with `'Address ID: ' . $item['id']`, though it needs to be translation-ready using `__()`, and let's use `sprintf()`).

Then, move the old `$name` assignment inside an if statement that checks `if ( ! empty( $customer ) ) {`

So that way, we have a default using the address ID at the very start, then we only use the customer name or email if the `$customer` variable is not empty.

While we're here: IMO the `isset( $customer->name )` check should be changed to be `! empty()` instead.